src/main/java/com/ericsson/ei/frontend/utils/BackEndInstanceFileUtils.java

```diff
@@ -28,15 +32,25 @@
 
     private String eiInstancesPath;
 
+    @Autowired
+    BackEndInstancesUtils backendInstancesUtils;
+    
     @Value("${ei.backendInstancesFilePath:#{null}}")
     private String backendInstancesFilePath;
+    
+    @Value("${ei.backendInstancesListJsonContent:#{null}}")
+    private String backendInstancesListJsonContent;
 
     @PostConstruct
     public void init() throws IOException {
         LOG.info("Initiating BackEndInstanceFileUtils.");
 
+        JsonArray backendInstancesListJsonArray = (JsonArray) new JsonParser().parse(backendInstancesListJsonContent.toString());
```

Comment:
This init() method has become quite large. It may be time to break the init method into smaller parts.

Reply:
Fixed.